### PR TITLE
test(Integration_test): add integration test to the CircleCI workflow

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Description
`Automated the process of running the integration tests to trigger the CircleCI workflow. Below are the changes made:`
`- integrated CircleCI workflow with the integration tests in the Contile repo to trigger the workflow`
`- added a `docker-compose.yml` file with the image details for Partner, Client and Contile`
`- added `/volumes` folder where the test `scenarios.yml`, `responses.yml` and `adm_settings.json` files resides required to run the tests`
`- updated the CircleCI `config.yml` file with the necessary steps to run the integration tests with commands`

## Testing

@mozilla-services/context-services - Please review the files added together with the commands. Thanks!

### CODE REVIEW

**Review by jrconlin — APPROVED** [review]
LGTM. (Wondering if I should remove the "Fake-Response" / test mode flags from the tile code)

File: .circleci/config.yml [hunk 1]
```diff
@@ -186,13 +216,28 @@ workflows:
...
               only: /.*/
+          requires:
+            - build
+      - contile-integration-tests:
+          name: contile-integration-tests
+          requires:
+          - build
+          filters:
+            branches:
+              only:
```
Comment (pjenvey) [inline .circleci/config.yml:235]: Sounds like this won't add much time to CI so let's run this everywhere like the other targets. It'll be nice to have it catch issues on pull requests.

**Review by pjenvey — CHANGES_REQUESTED** [review]

**Comment by jrconlin** [comment]
I don't think we need it currently, but I wonder if we should use something like [`store_test_results`](https://circleci.com/docs/2.0/configuration-reference/#storetestresults) if we're running tests like these? It may be overkill.

File: .circleci/config.yml (hunk 1 above)
Comment (ashrivastava-qa) [reply .circleci/config.yml]: okay. Made the changes. Thanks for the r+!

**Review by jrconlin — APPROVED** [review]

**Review by pjenvey — APPROVED** [review]

**Comment by jrconlin** [comment]
By the way, THANK YOU for doing this! This is a huge help.

**Comment by pjenvey** [comment]
> I don't think we need it currently, but I wonder if we should use something like [`store_test_results`](https://circleci.com/docs/2.0/configuration-reference/#storetestresults) if we're running tests like these? It may be overkill.

That does look like a nice feature (particularly the test insights):

https://circleci.com/docs/2.0/collect-test-data/

We'd need the integration test's pytest runner to output `--junitxml=out.xml`. Also the docker compose would probably need a volume mount letting it write out that `out.xml`

**Comment by pjenvey** [comment]
Oh but the Insights is for the self hosted CircleCI Server only though

**Comment by jrconlin** [comment]
Yeah, I think it just needs the workspace to compile the report, but I'm not sure. 

Something to play around with in the future, I suppose.

**Comment by ashrivastava-qa** [comment]
Yeah, I think that's a good suggestion @jrconlin and @pjenvey. I'll try and see if I can find out more about it to be implemented. Thanks! :)

**Pull request merged**